backlog-set-due-date

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
+
+dry_run="${dry_run:-"0"}"
+project_id="${project_id:-18}"
+ticket_limit="${ticket_limit:-200}"
+host="${host:-"https://progress.opensuse.org"}"
+issues=$(mktemp)
+
+
+[ "$dry_run" = "1" ] && prefix="echo"
+
+if [ "$dry_run" != "1" ]; then
+  redmine_api_key="${redmine_api_key:?"Need redmine API key"}"
+
+  curl -s -H "X-Redmine-API-Key: $redmine_api_key" "$host/issues.json?project_id=$project_id&limit=$ticket_limit" | jq -r \
+  '.issues | .[] |  select(.priority.name!="Low" and .due_date==null and .assignee!=null and .status.name!="Blocked")' >"$issues"
+else
+  if [ $# -ne 1 -o ! -f $1 ]; then
```

Comment:
`./backlog-set-due-date: line 18: [: argument expected`
```suggestion
  if [ $# -ne 1 ] || [ ! -f "$1" ]; then
```

Reply:
Ok that worked on my previous test but yes, changing. the shellcheck says too